templates/admin/index.html.twig

```diff
@@ -1,30 +1,21 @@
-<title>{% block title %}Page Admin -{% endblock %}</title>
+<title>
+	{% block title %}Page Admin -
+	{% endblock %}
+</title>
 {% block stylesheet %}
-<link
-    href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
-    rel="stylesheet"
-/>
-<link
-    href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
-    rel="stylesheet"
-/>
-<link
-    rel="stylesheet"
-    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
-/>
-{{ encore_entry_link_tags("adminEntry") }}
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"/>
```

Comment:
those links will be imported on every admin page ? Shift'em into admin layout (admin base html)